### PR TITLE
LRDOCS-5620 forms links and screenshots

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-5620 @natocesarrego this is just some checking of screenshots and links, to make sure that the published docs match what's in the release the release bundle, and have working links. There's also a note we added on form translations, which I mentioned in the team slack channel. @nealrobbins, thanks for doing this!

### CODE REVIEW

File: discover/portal/articles/130-forms/10-workflow-and-forms.markdown [hunk 1]
```diff
@@ -9,7 +9,7 @@ instance or Site level.
 
 Forms is different, and that's why its assets don't appear in the above image.
 There are so many use cases for forms, and there could be so many per site, that
-a site---or instance---scoped workflow configuration won't serve your needs 
+a site- or instance-scoped workflow configuration won't serve your needs
```
Comment (natocesarrego) [inline discover/portal/articles/130-forms/10-workflow-and-forms.markdown]: Should have a blank character after the first hyphen?

Comment (rbohl) [reply discover/portal/articles/130-forms/10-workflow-and-forms.markdown]: @natocesarrego yeah, that's a shorthand way of saying

"a site-scoped or instance-scoped workflow configuration..."

**Comment by rbohl** [comment]
I'll fix the broken links! Thanks!

File: discover/portal/articles/130-forms/10-workflow-and-forms.markdown (hunk 1 above)
Comment (natocesarrego) [reply discover/portal/articles/130-forms/10-workflow-and-forms.markdown]: Nice. Thanks for the explanation! 🙂

**Comment by natocesarrego** [comment]
No problem, @rbohl. I've just seen your email regarding the docs sync. Thanks for the update.

**Pull request merged**